baseplate/context/memcache/lib.py

```diff
@@ -44,13 +44,16 @@ class Flags(object):
 
 
 def decompress_and_load(key, serialized, flags):
-    """Deserialization method compatible with dump_and_compress().
+    """Modern deserialization method.
```

Comment:
Agreed that it was obtuse. I've tried to clean it up with a bit more explicit definition of when you'd want the "legacy" style and I'm just making the "modern" style describe itself as _the_ way to do serde. How's this look now?